lib/services/local/sessiontracker.go

```diff
@@ -116,16 +117,46 @@ func (s *sessionTracker) GetActiveSessionTrackers(ctx context.Context) ([]types.
 		return nil, trace.Wrap(err)
 	}
 
-	sessions := make([]types.SessionTracker, len(result.Items))
-	for i, item := range result.Items {
+	sessions := make([]types.SessionTracker, 0)
+	expired := make([]backend.Item, 0)
+	now := time.Now().UTC()
+	for _, item := range result.Items {
 		session, err := unmarshalSession(item.Value)
 		if err != nil {
 			return nil, trace.Wrap(err)
 		}
 
-		sessions[i] = session
+		switch {
+		case session.Expiry().After(now):
+			// Keep any items that aren't expired.
+			sessions = append(sessions, session)
+		case item.Expires.IsZero():
```

Comment:
```suggestion
		case session.Expiry().IsZero():
```

Reply:
This doesn't check the backend expiry which is what matters here